setup.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+from setuptools import setup, find_packages
+
+setup(name='avalon_sdk',
+      version='0.0.4',
+      description='Avalon SDK ',
+      author='Hyperledger Avalon ',
+      author_email="karthika.murthy@intel.com",                                               
```

Comment:
We don't usually keep Intel email `author_email` in setup.py. I'm not sure what's Intel recommendation here. Can you please find out.